src/components/Button.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import Button from './Button';
+import { render, screen } from '@testing-library/react';
+import fireEvent from '@testing-library/user-event';
+import { ReactComponent as IconLarge } from '../images/play-large.svg';
+
+test('check if any text is present (it should not) if you pass the iconOnly and icon props.', () => {
+  const clickPlay = jest.fn();
+  render(
+    <Button onClick={clickPlay} icon={IconLarge} iconOnly children="Play" />
```

Comment:
You can just pass `jest.fn()` or even `() => null` if you don't test `onClick`'s behavior